staging_service/autodetect/GenerateMappings.py

```diff
@@ -1,27 +1,6 @@
 """
-* This script generates various potential api responses and serves as a list of supported apps and extensions
```

Comment:
Is there a reason all this documentation is deleted? it seems valuable

Reply:
This was a copypasta error. Thanks!